msl_expressions/autogenerated/src/Plans/Standards/Own/ThrowIn/constraints/ThrowInNearGoal1461237603689Constraints.cpp

```diff
@@ -270,4 +271,8 @@ namespace alicaAutogenerated
 
 // State: Wait
 
+// State: timeoutForPull
+
```

Comment:
Did adding this state improve anything? Commit-messages look ambiguous.

Reply:
This is an autogenerated part. So somebody probably just forget to commit this file, after he commited a new state in the ThrowInNearGoal Plan.

Reply:
But which version of the plan is the better one? With this state or without? The state was added during the tournament and the duration in the transitions has been modified a lot over some commits, so i'm not sure if it's an actual improvement or just an experiment.

Reply:
Update: Stopfer said keep it